SystemSettings.php

```diff
@@ -18,6 +18,7 @@
 use Piwik\SettingsPiwik;
 use Piwik\Tracker\Cache;
 use Piwik\Validators\Email;
+use Piwik\Network\IPUtils;
```

Comment:
@elvmp I had to change this import in-order, else the code cannot find the namespace
```suggestion
use Matomo\Network\IPUtils;
```